tutorials/algorithms/05_qaoa.ipynb

```diff
@@ -6,7 +6,7 @@
    "source": [
```

Comment:
<pre class="ql-syntax" spellcheck="false">def objective_value(x: np.ndarray, w: np.ndarray) -&gt; float:
&nbsp; &nbsp; """Compute the value of a cut.
&nbsp; &nbsp; Args:
&nbsp; &nbsp; &nbsp; &nbsp; x: Binary string as numpy array.
&nbsp; &nbsp; &nbsp; &nbsp; w: Adjacency matrix.
&nbsp; &nbsp; Returns:
&nbsp; &nbsp; &nbsp; &nbsp; Value of the cut.
&nbsp; &nbsp; """
&nbsp; &nbsp; X = np.outer(x, (1 - x))
&nbsp; &nbsp; w_01 = np.where(w != 0, 1, 0)
&nbsp; &nbsp; return np.sum(w_01 * X)
</pre><p>This function was already defined at the start of the notebook, but here it's explained. I think it makes more sense to have the explained version at the start or have both versions with the explanation/docstring.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1372/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='10'/>